chart/static-files/openapi.json

```diff
@@ -1895,13 +1895,13 @@
                   "empty-config": {
                     "summary": "The config parameter is empty.",
                     "value": {
-                      "error": "Parameters 'config' and 'split' are required"
+                      "error": "Parameter 'config' is required"
                     }
                   },
                   "empty-split": {
                     "summary": "The split parameter is empty.",
                     "value": {
-                      "error": "Parameters 'config' and 'split' are required"
+                      "error": "Operation not supported or not found"
```

Comment:
why is it different from "empty-config" ?

Reply:
Given the new logic of endpoints responding based on the params to its corresponding processing steps we have 
- Infer the input type from the query params
- Based on it, get the result from the processing step list
For this specific case I had to modify the response of /first-rows case with empty split since as per our configuration on
            "/first-rows": {"split": ["/first-rows"]},
we only support first-rows passing dataset, config and split params, if we pass only dataset and config, the process would assume that the client is trying to request /first-rows only config  input_type which is not supported.
I know that maybe a better message would be "split" param missing but what if in the future we support /first-rows with both input_types (config and split) , that message will never be shown.